docker/entrypoint.sh

```diff
@@ -1,6 +1,9 @@
+#!/bin/bash
+export ENV_MAX_HEAP_SIZE=$2
+export ENV_INI_HEAP_SIZE=$3
 python3 wren-sqlglot-server/main.py &
 
 # Required add-opens=java.nio=ALL-UNNAMED for Apache arrow in the Snowflake
-java -Dconfig=etc/config.properties \
+java -Xmx${ENV_MAX_HEAP_SIZE:-"512m"} -Xms${ENV_INI_HEAP_SIZE:-"64m"}  -Dconfig=etc/config.properties \
```

Comment:
Curious, why use `INI` not `MIN`?

Reply:
Initial heap size.

Reply:
hmm.. maybe change to `MIN` would be more clear.